utility/mm2_crawler

```diff
@@ -60,7 +67,17 @@ class InjectingFilter(logging.Filter):
         self.thread_id = thread_id
 
     def filter(self, record):
-        return threadlocal.thread_id == self.thread_id
+        try:
+            return threadlocal.thread_id == self.thread_id
```

Comment:
No, but if the shutdown signal comes in too early, before the ```thread_id``` is set, it backtraces here. This is just to make sure early signals are handled properly.